crates/vm-superio-ser/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "vm-superio-ser"
+version = "0.1.0"
```

Comment:
I think we sort of missed talking about this during the design phase, but how do we think about versions? We should have a way to specify what version of vm-superio-ser is compatible with what version of vm-superio.

Reply:
Added a new section in the README, hopefully it makes sense.